lib/gui/app.js

```diff
@@ -219,3 +220,29 @@ app.controller('HeaderController', function(SelectionStateModel, OSOpenExternalS
   };
 
 });
+
+app.controller('ApplicationController', function(SupportedFormatsModel, OSDialogService, AnalyticsService, SelectionStateModel) {
+
+  /**
+   * @summary Select image
+   * @function
+   * @public
+   *
+   * @param {Object} image - image
+   *
+   * @example
+   * OSDialogService.selectImage()
+   *   .then(ApplicationController.selectImage);
+   */
+  this.selectImage = (image) => {
```

Comment:
Once this change has been made, we can change to analytics message to "Drop image".